package.json

```diff
@@ -73,7 +73,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.8.0",
     "@qwant/map-style-builder": "github:Qwant/map-style-builder#9808391",
-    "@qwant/qwant-basic-gl-style": "^1.1.0",
+    "@qwant/qwant-basic-gl-style": "^1.3.1",
```

Comment:
You have to do it using `npm install` so that the `package-lock.json` file is kept up-to-date.

Reply:
Yes. And I would expect `npm ci` to catch this. One more inconsistency from `npm` I suppose...